.github/workflows/image.yaml

```diff
@@ -16,21 +16,15 @@
 name: image
 
 on:
-  pull_request:
-    types:
-      - opened
-      - synchronize
-    branches:
-      - main
-      - release-*
   push:
     branches:
       - main
+      - "pull-request/[0-9]+"
```

Comment:
Can you rebase? I believe this should already be updated on main.

Reply:
Rebased. I've also chosen to maintain lexical order here